persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java

```diff
@@ -565,6 +559,36 @@ private void authorizeAlterSchemaStatement(
         tableName);
   }
 
+  private void authorizeSearchIndexStatement(
```

Comment:
the unit test was not extended to test the `authorizeSearchIndexStatement` path, can you provide one?